client-java/controller/pom.xml

```diff
@@ -154,6 +154,39 @@
             <artifactId>mockito-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <!--Need to confirm with Andrea-->
+            <!-- check if this version configuration is needed to add to the root pom.xml-->
+            <version>8.0.23</version>
+            <scope>test</scope>
```

Comment:
yep, those should be in root pom file

Reply:
why are those `<version>` still here? 
they should had been declared in `<dependencyManagment>` in root pom file, so child modules do not need to define any `<version>`.

Reply:
thanks. these are updated with the latest commits.